exasol/slc/internal/tasks/export/export_container_base_task.py

```diff
@@ -35,31 +38,35 @@
 
 
 class ExportContainerBaseTask(FlavorBaseTask):
-    logger = logging.getLogger("luigi-interface")
-    export_path = luigi.OptionalParameter(None)
-    release_name = luigi.OptionalParameter(None)
-    release_goal = luigi.Parameter(None)
+    task_logger = logging.getLogger("luigi-interface")
```

Comment:
Needed to rename because there was clash with `BaseTask.logger`

Reply:
The question is actually, why do we redefine it at all

Reply:
Good question, it's there since the initial version of this repo:
https://github.com/exasol/script-languages-container-tool/blob/2e3fc8ff13b8df159174cf7fa7e69009f9c8a20e/exasol_script_languages_container_tool/lib/tasks/export/export_container_base_task.py

As the logger is only used to print `info` message, my guess is that, the logs were used during the initial development cycle to print debug messages. 

I believe it's ok to use the BaseTask.logger.